UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment.UnitTests/Helpers/FssApiClientTests.cs

```diff
@@ -188,7 +188,7 @@ public void DoesDownloadFile_Returns_OK()
 
             _fakeFssApiClient = new FssApiClient(_fakeHttpClientFactory);
 
-            Task<HttpResponseMessage>? result = _fakeFssApiClient.DownloadFile("http://test.com", _authToken);
+            Task<HttpResponseMessage>? result = _fakeFssApiClient.DownloadFile("http://test.com", _authToken, "bytes=1-1024");
```

Comment:
please can we use fluent assertion

Reply:
We have utilised Assert of Nunit throughout unit testing. From the subsequent project on, fluid assertion will be implemented.